### PR TITLE
Enable CLI installation on libreoffice

### PR DESCRIPTION
**Pull request by cbookg**
Changes include (mainly for alpine Linux):
- Using APK package manager instead of APT.
- Installing gcsfuse from source as it is not available for APK.

### CODE REVIEW

File: startupscript/gcp/resource-mount.sh [hunk 1]
```diff
@@ -11,21 +11,35 @@
...
 
-  # install packages needed to install gcsfuse
-  apt-get install -y \
-    fuse \
-    lsb-release
+    apk add --no-cache fuse git go
 
-  # Install based on gcloud docs here https://cloud.google.com/storage/docs/gcsfuse-install.
-  GCSFUSE_REPO="gcsfuse-$(lsb_release -c -s)"
-  readonly GCSFUSE_REPO
```
Comment (mbookman) [inline startupscript/gcp/resource-mount.sh:23]: Can you add a reference to where these install steps come from? Is there a doc like was referenced in the existing comment.

**Review by mbookman — CHANGES_REQUESTED** [review]
LGTM. One small question.

File: startupscript/gcp/resource-mount.sh (hunk 1 above)
Comment (cbookg) [reply startupscript/gcp/resource-mount.sh:23]: Oh its from the same page, but the comment is moved below into the "apt" section.  I can put it at the top or soemthing.

Comment (cbookg) [reply startupscript/gcp/resource-mount.sh:23]: updated comment

**Review by mbookman — APPROVED** [review]
LGTM

**Review by mbookman — APPROVED** [review]
LGTM

**Pull request merged**